src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1938,6 +1966,251 @@
+  "com.minecolonies.research.circus.name": "Circus",
+  "com.minecolonies.research.compost.name": "Compost",
+  "com.minecolonies.research.compress.name": "Compress",
+  "com.minecolonies.research.diamondcoated.name": "Diamond Coated",
+  "com.minecolonies.research.diamondskin.name": "Diamond Skin",
+  "com.minecolonies.research.diligent.name": "Diligent",
+  "com.minecolonies.research.dodge.name": "Dodge",
+  "com.minecolonies.research.dodge.subtitle": "If you can dodge a wrench...",
+  "com.minecolonies.research.duelist.name": "Duelist",
+  "com.minecolonies.research.dung.name": "Dung",
+  "com.minecolonies.research.epicure.name": "Epicure",
+  "com.minecolonies.research.evasion.name": "Evasion",
+  "com.minecolonies.research.evasion.subtitle": "Duck",
+  "com.minecolonies.research.fear.name": "Fear",
+  "com.minecolonies.research.feint.name": "Feint",
+  "com.minecolonies.research.fertilizer.name": "Fertilizer",
+  "com.minecolonies.research.festival.name": "Festival",
+  "com.minecolonies.research.fullretreat.name": "Full Retreat",
+  "com.minecolonies.research.fullstock.name": "Full Stock!",
+  "com.minecolonies.research.glutton.name": "Glutton",
+  "com.minecolonies.research.goodveins.name": "Good Veins",
+  "com.minecolonies.research.gorger.name": "Gorger",
+  "com.minecolonies.research.gourmand.name": "Gourmand",
+  "com.minecolonies.research.growth.name": "Growth",
+  "com.minecolonies.research.hardened.name": "Hardened",
+  "com.minecolonies.research.healingcream.name": "Healing Cream",
+  "com.minecolonies.research.heavymachinery.name": "Heavy Machinery",
+  "com.minecolonies.research.hormones.name": "Hormones",
+  "com.minecolonies.research.improveddodge.name": "Improved Dodge",
+  "com.minecolonies.research.improveddodge.subtitle": "Dip",
+  "com.minecolonies.research.improvedevasion.name": "Improved Evasion",
+  "com.minecolonies.research.improvedevasion.subtitle": "Dive",
+  "com.minecolonies.research.improvedleather.name": "Improved Leather",
+  "com.minecolonies.research.ironarmor.name": "Iron Armor",
+  "com.minecolonies.research.ironskin.name": "Iron Skin",
+  "com.minecolonies.research.isthisredstone.name": "Is This Redstone?",
+  "com.minecolonies.research.knighttraining.name": "Knight Training",
+  "com.minecolonies.research.lightning.name": "Lightning",
+  "com.minecolonies.research.madness.name": "Madness!",
+  "com.minecolonies.research.magiccompost.name": "Magic Compost",
+  "com.minecolonies.research.master.name": "Master",
+  "com.minecolonies.research.masterbowman.name": "Master Bowman",
+  "com.minecolonies.research.masterswordsman.name": "Master Swordsman",
+  "com.minecolonies.research.memoryaid.name": "Memory Aid",
+  "com.minecolonies.research.morebooks.name": "More Books",
+  "com.minecolonies.research.motherlode.name": "Motherlode",
+  "com.minecolonies.research.multishot.name": "Multishot",
+  "com.minecolonies.research.multishot.subtitle": "Known side effects include double vision.",
```

```suggestion
  "com.minecolonies.research.multishot.subtitle": "Known side effects include double vision double vision.",
```
Double it double it!